doc/source/guide/acquiring_data/fido.rst

```diff
@@ -0,0 +1,73 @@
+---------------------------------
+Finding and Downloading Data using Fido
+---------------------------------
+
+This guide outlines how to search for and download data using SunPy's
+Federated Internet Data Obtainer...or more usually (and
+sanely) referred to as Fido.  Fido is a unified interface for seeking
+and downloading solar physics data irrespective of the underlining
+client or webservice through which the data is obtained, e.g. VSO,
+JSOC, etc.  It therefore supplies a single, easy and consistent way to
+obtain most forms of solar physics data.
+
+Import
+-----
+
+SunPy's Fido module is in ``sunpy.net``.  It can be imported as follows::
+
+    >>> from sunpy.net import Fido
```

Comment:
👎 this should be `from sunpy.net import Fido, attrs as a` we should always use the same import and `Fido` is useless without `attrs`